src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,64 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { useEffect, useState } from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  handleFiltrationQueries: (finishQuery?: string, searchQuery?: string) => void;
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+export const TodoFilter: React.FC<Props> = ({ handleFiltrationQueries }) => {
+  const [finishQuery, setFinishQuery] = useState('all');
```

Comment:
It's not a string, it should have only a few valid values

Reply:
![image](https://github.com/user-attachments/assets/fac7d5b1-c711-48cd-a9a1-6f9b9c7d99cf)

it's completely string